tools/ui-components/src/drop-down/drop-down-button/drop-down-toggle.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCaretDown, faCaretUp } from '@fortawesome/free-solid-svg-icons';
+import { Menu } from '@headlessui/react';
+
+type MenuProps = React.ComponentPropsWithoutRef<typeof Menu>;
+
+export type DropDownButtonProps = MenuProps & {
+  block?: boolean;
+  dropup?: boolean;
+  bsStyle?: 'primary';
+};
+
+const defaultClassNames = [
+  'cursor-pointer',
+  'border-3',
+  'text-center',
+  'touch-manipulation',
+  'bg-background-quaternary',
+  'text-foreground-secondary',
+  'px-3',
+  'py-1.5',
+  'relative',
+  'hover:bg-foreground-primary',
+  'hover:text-background-primary'
+];
+
+export function DropDownButton({
+  children,
+  block,
+  dropup,
+  bsStyle,
+  className,
+  ...rest
+}: DropDownButtonProps): JSX.Element {
+  if (block) defaultClassNames.push('btn-block');
+  if (bsStyle) defaultClassNames.push('border-foreground-primary');
```

Comment:
Personally, I think this is better deleted, as far as I know which may be wrong, it's always true so instead of making the user ask a question that is always true, we can reduce the load on the use by keeping it adding it to `defaultClassNames`.

```suggestion
  if (block) defaultClassNames.push('btn-block');
  if (bsStyle) defaultClassNames.push('border-foreground-primary');
```

Reply:
Sure, as long as we are getting the same look as the dropdown in the learning platform.